core/jobs/batch_jobs/reject_invalid_suggestion_and_delete_invalid_translation_jobs.py

```diff
@@ -0,0 +1,596 @@
+            self.pipeline)
+        deletion_result_dicts = (
+            entity_translation_models
+            | 'Get deletion results' >> beam.ParDo(
+                    self.DeleteTranslationsWithInvalidContentIds())
+            | 'Filter out None values' >> beam.Filter(lambda x: x is not None)
+        )
+
+        deleted_translations_count_job_run_results = (
+            deletion_result_dicts
+            | 'Deleted translations counts' >> beam.Map(
+                    lambda x: x['deleted_translations_count'])
+            | 'Total deleted translations count' >> (
+                beam.CombineGlobally(sum))
+            | 'Only create result for non-zero number of objects' >> (
+                beam.Filter(lambda x: x > 0))
+            | 'Report total deleted translations count' >> beam.Map(
+                lambda result: (
+                    job_run_result.JobRunResult.as_stdout(
+                        f'DELETED TRANSLATIONS COUNT SUCCESS: {result}'
+                    )))
+        )
+
+        updated_entity_translation_models = (
+            deletion_result_dicts
+            | 'Updated entity translation models' >> beam.Map(
+                    lambda x: x['entity_translation_model'])
+        )
+
+        updated_entity_translation_models_count_job_run_results = (
+            updated_entity_translation_models
+            | 'Updated entity transltion models count' >> (
+                job_result_transforms.CountObjectsToJobRunResult(
+                    'UPDATED ENTITY TRANSLATION MODELS COUNT'))
+        )
+
+        latest_version_updated_entity_translation_models = (
+            updated_entity_translation_models
+            # PCollection<entity_id: entity_translation_model>.
+            | 'Add entity id as key' >> beam.WithKeys(  # pylint: disable=no-value-for-parameter
+                lambda model: model.entity_id)
+            # PCollection<entity_id: list(entity_translation_model)>.
+            | 'Group by entity id' >> beam.GroupByKey()
+            # PCollection<entity_id: entity_translation_model>.
+            | 'Filter model with latest entity version' >> beam.ParDo(
+                self.GetLatestModel())
+            # PCollection<entity_translation_model>.
+            | 'Get list of latest entity transaltion model' >> beam.Values()  # pylint: disable=no-value-for-parameter
+        )
+
+        updated_exp_opportunity_models = (
+            latest_version_updated_entity_translation_models
+            | 'Get updated explortion opportunity models' >> beam.ParDo(
+                    self.ComputeUpdatedExpOpportunityModel())
+        )
+
+        unused_put_results = (
+            (
+                updated_entity_translation_models,
+                updated_exp_opportunity_models
+            )
+            | 'Merge lists' >> beam.Flatten()
+            | 'Put models into the datastore' >> ndb_io.PutModels()
+        )
+
+        return (
```

Comment:
Could you please output the same data here as you do in the audit job, for debugging purposes in case something goes wrong in production? This will happen automatically if you reuse as much of the functionality as possible between the jobs.

Reply:
done